.travis.yml

```diff
@@ -10,12 +10,18 @@ before_install:
 install:
 -  npm install
 -  npm install -g grunt-cli
+-  pip install .
 -  pip install -r requirements_dev.txt
 -  pip install coveralls
 script:
+# Test that we can run from source directory
 - bin/kalite start --traceback -v 2
 - bin/kalite status
 - bin/kalite stop --traceback -v 2
+# Test that we can run as the installed script
```

Comment:
It's a test that they don't return with a non-zero failure. I'm contemplating doing a Tox.ini to have more clarity. Is it the word "test" that annoys you because it's not a TestCase instance? :)